### PR TITLE
Score details page components, summary page UI

### PR DESCRIPTION
**Pull request by kelsiliul**
Finished Components, and summary details UI.

Score Box,
image
Score Details Screen,
image
Score Summary,
image

### CODE REVIEW

File: app/src/main/java/com/cornellappdev/score/components/ScoreSummary.kt [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.score.model.ScoreEvent
+import com.cornellappdev.score.theme.Style.bodyNormal
+import com.cornellappdev.score.theme.Style.heading5
+import com.cornellappdev.score.util.scoreEvents1
+
+
+@Composable
+fun ScoringSummary(scoreEvents: List<ScoreEvent>) {
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/score/components/ScoreSummary.kt:24]: Please add a 

`modifier: Modifier = Modifier` parameter here and use that as the modifier for the lazy column. It looks like the horizontal padding here is none, which is fine--we just need to make sure people in the future can add padding if needed on the actual screen. Adding a modifier allows this.

File: app/src/main/java/com/cornellappdev/score/components/ScoreSummary.kt [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                    textAlign = TextAlign.Center
+                )
+                Text(
+                    text = " - ",
+                    style = heading5,
+                    textAlign = TextAlign.Center
+                )
+                Text(
+                    text = awayScore.toString(),
+                    style = if (awayScore >= homeScore) heading5.copy(fontWeight = FontWeight.Bold) else heading5,
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/score/components/ScoreSummary.kt:89]: Prob just make this `>`, right? If it ties, this claims the away wins.

File: app/src/main/java/com/cornellappdev/score/model/Game.kt [hunk 3]
```diff
@@ -7,5 +7,60 @@ data class GameCardData(
...
+
+// Individual scoring event details for the scoring summary
+data class ScoreEvent(
+    val id: Int,             // Unique identifier for each event
+    val time: String,        // Event time (e.g., "6:21")
+    val quarter: String,     // Quarter (e.g., "1st Quarter")
+    val team: Team,          // Team object that includes name and logo URL
+    val eventType: String,   // Event type (e.g., "Field Goal", "Touchdown")
+    val score: String,        // Current score after the event (e.g., "10 - 7")
+    val description: String? = null )
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/score/model/Game.kt:33]: Plz delete these comments 😅

Comment (thisjustin123) [reply app/src/main/java/com/cornellappdev/score/model/Game.kt:33]: (The others ABOVE each data class are fine; plz convert those to javadoc comments actually to make them useful!)

File: app/src/main/java/com/cornellappdev/score/screen/GameDetailsScreen.kt [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.score.R
+import com.cornellappdev.score.model.ScoreEvent
+import com.cornellappdev.score.model.Team
+
+@Composable
+fun GameDetailsScreen(scoreEvents: List<ScoreEvent>, onArrowClick: () -> Unit) {
+    Column(
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/score/screen/GameDetailsScreen.kt:26]: I assume this is still missing some stuff before it's the full screen? Add a TODO if you're not planning to finish that in this PR!

File: app/src/main/java/com/cornellappdev/score/screen/GameScoreSummaryScreen.kt [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+fun GameScoreSummaryScreenDetail(scoreEvents: List<ScoreEvent>) {
+    LazyColumn(
+        modifier = Modifier.fillMaxWidth()
+    ) {
+        items(scoreEvents.size) { event ->
+            ScoreEventItemDetailed(event = scoreEvents[event])
+            Divider(color = Color.LightGray, thickness = 0.5.dp)
+        }
+    }
+}
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/score/screen/GameScoreSummaryScreen.kt:89]: Plz move this to the top of the file!

File: app/src/main/java/com/cornellappdev/score/components/ScoreSummary.kt (hunk 2 above)
Comment (kelsiliul) [reply app/src/main/java/com/cornellappdev/score/components/ScoreSummary.kt:89]: I interpreted wrong, the special font style depends on which team is scoring. now fixed.

**Pull request merged**